sig-security/joint-review.md

```diff
@@ -1,414 +1,414 @@
+
+As WebAssembly is increasingly used in Edge Computing scenarios where it is difficult to deploy Linux containers or when the application performance is vital, KubeEdge provides a lightweight alternative to Linux containers in edge native environments by integrating with **WasmEdge**. WasmEdge is a lightweight, high-performance, and extensible WebAssembly runtime for cloud native, edge, and decentralized applications. It powers serverless apps, embedded functions, microservices, smart contracts, and IoT devices.
+
+For the images registry infrastructure, KubeEdge can work with **Harbor** to manage images efficiently. Harbor is an open source trusted cloud native registry project that stores, signs, and scans content. Harbor extends the open source Docker Distribution by adding the functionalities usually required by users such as security, identity and management. Having a registry closer to the build and run environment can improve the image transfer efficiency.
+
+#### Vendors and products
+
+The KubeEdge project is also working with many other vendors to integrate KubeEdge into their solutions to enrich the Cloud Native Edge Computing ecosystem.
+
+EdgeStack, an intelligent edge computing platform of HarmonyCloud, builds a edge-cloud synergy system through the integration with KubeEdge, which can support access of millions of edge nodes and devices. It is designed for large-scale, massive access, low bandwidth, low latency, high performance, and high stability of edge computing.  At present, it has landed in communications, transportation, finance, and many other fields.
+
+The DaoCloud Edge Computing platform uses KubeEdge to manage edge applications. With cloud-edge synergy, this platform responds to edge application requests in real time, and constantly monitors node, device, and application status. The cloud is responsible for edge node registration, management, and application and configuration delivery. At the edge runs edge applications to which edge autonomy is available. For devices, multi-protocol access is supported and standard APIs are provided to connect to devices.
+
+Based on KubeEdge, Intelligent EdgeFabric (IEF) of Huawei Cloud embeds cloud native into edge computing, supporting ultimate lightweight deployments, edge intelligence, and powerful computing power. It has been widely used in smart campus, industrial quality inspection, mining, and smart transportation where the collaboration between edge and cloud yields brilliant results. 
+
+QingCloud KubeSphere Enterprise Container Platform (QKCP), is an enterprise container management platform from KubeSphere open source community. Based on KubeEdge, applications and workloads are uniformly distributed and managed on the cloud and edge nodes, meeting the needs for application delivery, O&M, and management on a large number of edge and device devices. 
```

Comment:
`QingCloud KubeSphere Enterprise Container Platform (QKCP)` is now renamed to `KubeSphere Enterprise (KSE)`